src/libsystemd/sd-netlink/netlink-types-genl.c

```diff
@@ -205,7 +205,7 @@ static const NLType genl_wireguard_peer_types[] = {
         [WGPEER_A_FLAGS]                         = { .type = NETLINK_TYPE_U32 },
         [WGPEER_A_PRESHARED_KEY]                 = { .type = NETLINK_TYPE_BINARY, .size = WG_KEY_LEN },
         [WGPEER_A_PERSISTENT_KEEPALIVE_INTERVAL] = { .type = NETLINK_TYPE_U16 },
-        [WGPEER_A_ENDPOINT]                      = { .type = NETLINK_TYPE_SOCKADDR },
+        [WGPEER_A_ENDPOINT]                      = { .type = NETLINK_TYPE_SOCKADDR, .size = sizeof(struct sockaddr_in) },
```

Comment:
This takes sockaddr_in or sockaddr_in6. Please drop the change.

Reply:
Dropped